packages/block-library/src/heading/deprecated.js

```diff
@@ -48,7 +52,52 @@ const migrateCustomColors = ( attributes ) => {
 	};
 };
 
+const migrateTextAlign = ( attributes ) => {
+	const { align, ...rest } = attributes;
+	return TEXT_ALIGN_OPTIONS.includes( align )
+		? { ...rest, textAlign: align }
+		: attributes;
+};
+
+const TEXT_ALIGN_OPTIONS = [ 'left', 'right', 'center' ];
+
 const deprecated = [
+	{
+		supports: {
+			className: false,
```

Comment:
It would be easier to review if those settings were in the same order as in `block.json` file. Just saying for future, no need to update :)

Reply:
I changed that as well since I addressed the other feedback. Will keep in mind for the future too :)